lib/goo/sparql/mixins/solution_lang_filter.rb

```diff
@@ -6,53 +6,41 @@ class  LanguageFilter
         attr_reader :requested_lang, :unmapped, :objects_by_lang
 
         def initialize(requested_lang: nil, unmapped: false, list_attributes: [])
+          @attributes_to_translate = [:synonym, :prefLabel]

```

Comment:
need to translate all the attributes, not only :synonym and :prefLabel

Reply:
is there a variable that contains all the variables?

Reply:
I got this error if i tried it with all variables 

![image](https://user-images.githubusercontent.com/35062242/235147292-bd2296d3-eee5-4c0a-afea-e1716d5504b2.png)

Reply:
fixed with https://github.com/ontoportal-lirmm/goo/pull/32/commits#:~:text=Fix%20the%20issue%20of%20undefined%20%27id%27%20of%20the%20language%20filter%20module